Source/SwiftLintFramework/Rules/VariableNameMinLengthRule.swift

```diff
@@ -1,27 +1,40 @@
 //
-//  VariableNameMinLengthRule.swift
+//  VariableNameMinLengthNewRule.swift
```

Comment:
VariableNameMinLengthNewRule?

Also, did you accidentally re-attribute this file to yourself?